src/AppBundle/Controller/TagController.php

```diff
@@ -32,6 +32,20 @@ public function indexAction()
         ));
     }
 
+    /**
+     * @Route("/tag.json", name="allTags")
+     * @param Request $request
+     * @return \Symfony\Component\HttpFoundation\JsonResponse
+     */
+    public function autoCompleteAction(Request $request)
```

Comment:
il faut utiliser isXmlHttpRequest pour les requetes ajax.

Reply:
https://symfony.com/doc/3.4/quick_tour/the_controller.html#getting-information-from-the-request